piet-wgsl/examples/winit/src/test_scene.rs

```diff
@@ -50,7 +50,7 @@ const N_CIRCLES: usize = 0;
 #[allow(unused)]
 pub fn render_svg(sb: &mut SceneBuilder, svg: &PicoSvg, print_stats: bool) {
     use crate::pico_svg::*;
-    let start = std::time::Instant::now();
+    //let start = std::time::Instant::now();
```

Comment:
I'd be tempted to just remove this line entirely.

Reply:
Done